### DISCUSSION TITLE
What do you think about my Imagefilter extension?

### DISCUSSION THREAD

**Discussion by PetersOtto** [datenstrom/community#979] [Ask a question] [answered]
Hello everyone!

I have developed two new extensions that can be used to modify images.
The extension »yellow-ImageFilterCollection« is only a plugin for the extension »yellow-ImageFilter«. This means that your own filters are retained if the »yellow-ImageFilter« needs to be updated ... 

but if you are interested, it is best to take a look at the pages of the extensions.

* [yellow-ImageFilter](https://github.com/PetersOtto/yellow-ImageFilter)
* [yellow-ImageFilterCollection](https://github.com/PetersOtto/yellow-ImageFilterCollection)

&#129395;

**Comment by PetersOtto** [comment, marked as answer]
I have implemented webp support in version 0.9.5.
The »Yellow Image« extension does not support webp. Is there anything against the webp format? Perhaps security problems?

**Reply by markseuffert** [reply]
The image formats GIF, JPEG, PNG and SVG are supported at the moment. WEBP did not provide much in my tests. It did not result in significant smaller file size, to be honest, I could not see that WEBP is 30% smaller than JPEG at the same image quality. If you do care about page loading speed, you will probably get better results by selecting a smaller image resolution. Potentially interesting image formats in my opinion are [AVIF](https://en.wikipedia.org/wiki/AVIF) and [JXL](https://en.wikipedia.org/wiki/JPEG_XL).

**Reply by PetersOtto** [reply]
Thank you for your feedback and the additional information. I will  take a closer look at the [AVIF](https://en.wikipedia.org/wiki/AVIF) and [JXL](https://en.wikipedia.org/wiki/JPEG_XL) image formats  . 
&#x1F44D;

**Comment by PetersOtto** [comment]
I have written another plugin/add on for my »ImageFilter« extension. This makes it possible to apply filters to images that are located on the `start-blog.html`. Of course you can also use the `.webp` format without filters. 

* [CatchImage](https://github.com/PetersOtto/yellow-catchImage/tree/main)

🥳

**Reply by bttrx** [reply]
Guess, you mean `blog-start.html`.

**Reply by PetersOtto** [reply]
Of course, thank you. &#128077;
